examples/tut-6-rpc/docker-compose.yml

```diff
@@ -1,6 +1,9 @@
 version: '3.4'
 x-server-base: &server-base
-  image: reachsh/reach-app-tut-6-rpc-server:latest
+  image: reachsh/rpc-server:latest
+  volumes:
+    - ./server/build:/app/build
```

Comment:
This might be a temporary thing, but I think that there should be a custom image generated by `./reach server` that includes this copying into the image rather than mounting it at runtime